tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "outDir": "lib/esm",
+    "module": "esnext",
+    "declaration": true,
+    "target": "es5",
+    "lib": ["es6", "dom", "es2016", "es2017"],
+    "jsx": "react",
+    "moduleResolution": "node",
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "esModuleInterop": true,
+    "noImplicitReturns": true,
+    "noImplicitThis": true,
+    "noImplicitAny": true,
+    "strictNullChecks": true,
+    "suppressImplicitAnyIndexErrors": true,
+    "allowSyntheticDefaultImports": true
+  },
+  "include": ["src"],
+  "exclude": ["node_modules"]
+}
```

Comment:
Missing new line at the end of the file

<img width="196" alt="image" src="https://user-images.githubusercontent.com/27439197/226929921-2950c8e3-5b61-437c-8d63-bb63af6d66b8.png">